langchain/package.json

```diff
@@ -529,7 +532,7 @@
     "release": "release-it --only-version --config .release-it.json",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest --testPathIgnorePatterns=\\.int\\.test.ts --testTimeout 30000",
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch --testPathIgnorePatterns=\\.int\\.test.ts",
-    "test:integration": "NODE_OPTIONS=--experimental-vm-modules jest --testPathPattern=\\.int\\.test.ts --testTimeout 100000",
+    "test:integration": "NODE_OPTIONS=--experimental-vm-modules jest --testPathPattern=assemblyai\\.int\\.test.ts --testTimeout 100000",
```

Comment:
You can just run `yarn test:single path/to/test`!

Reply:
My bad, it was hard to get the testing stuff to work. Thanks for the heads up!